action.yml

```diff
@@ -1,22 +1,29 @@
-name: 'Composite Run Steps Action Template'
+name: 'Url Status Check'
 
-description: 'Add description here'
+description: 'Pings a url and outputs the status code and content of the response'
 
 inputs:
-  input-1:
-    description: 'Add a description here'
+  url:
+    description: 'The url to ping.'
     required: true
+  fail-on-bad-status:
+    description: 'A flag that specifies whether or not to fail the action when a 400 or higher status code is returned. The expected values are true and false.'
+    required: false
+    default: 'false'
```

Comment:
What do you think about flipping this to default to true?  If I was using this in a workflow I think I might want it to fail by default.  Thoughts?

Reply:
That seems reasonable to me! I'll go ahead and make that change.